client/src/components/DogSelection.js

```diff
@@ -0,0 +1,186 @@
+  curInterested: {
+    preference: "Love",
+    time: "11:00 PM",
+  },
+  notes:
+    "Lorem Ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ",
+  othersInterested: [
+    {
+      name: "Stacy",
+      preference: "Like",
+      time: "12:05 PM",
+    },
+    {
+      name: "Molly",
+      preference: "Love",
+      time: "1:42 PM",
+    },
+  ],
+  active: false,
+};
+const dog2 = dog1;
+const dog3 = dog1;
+const dog4 = dog1;
+const dog5 = dog1;
+// dummy data
+const allDogs = { 1: dog1, 2: dog2, 3: dog3, 4: dog4, 5: dog5 };
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const DogWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  overflow: scroll;
+  height: 660px;
+  @media screen and (max-width: 750px) {
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  @media screen and (max-width: 350px) {
+    width: 100%;
+  }
+`;
+
+const ActiveWrapper = styled.div`
+  height: 630px;
+  width: 259px;
+  margin-left: 1vw;
+  margin-right: 1vw;
+  margin-bottom: 2vh;
+  padding: 10px 0px;
+  border-radius: 10px;
+  border: 3px solid #8dc442;
+  background: ${(props) => (props.active ? "rgba(141, 196, 66, 0.5)" : "white")};
+`;
+
+const Button = styled.button`
+  position: relative;
+  width: 216px;
+  height: 52px;
+  bottom: 0;
+  margin-top: 15px;
+
+  background: #8dc442;
+  border-radius: 10px;
+
+  font-family: Nunito;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 30px;
+  letter-spacing: 0em;
+
+  cursor: ${(props) => (props.cursor ? "pointer" : "default")};
+`;
+
+const Text = styled.p`
+  margin: 0;
+  margin-bottom: 15px;
+
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 36px;
+  letter-spacing: 0em;
+  text-align: center;
+`;
+
+function DogSummaryWrap({ dog, active, onClick }) {
+  return (
+    <ActiveWrapper onClick={onClick} active={active}>
+      <DogSummary props={dog} />
+    </ActiveWrapper>
+  );
+}
+
+const handleSubmit = (current) => {
+  if (current === -1) {
+    // nothing was submitted, so do nothing
+  } else {
+    // TODO: handle submit where allDogs[current] is the dog we are dealing with
+  }
+};
+
+function DogSelection() {
+  const [current, setCurrent] = useState(-1);
+
+  return (
+    <Content>
+      <Text>Select a dog to confirm Meet & Greet</Text>
+      {/* map out the dogs */}
+      <DogWrapper>
+        {Object.entries(allDogs).map(([key, dog]) => {
+          if (key === current) {
+            // element is active
+            return <DogSummaryWrap onClick={() => setCurrent(-1)} active dog={dog} />;
+          }
+          // element is not active
+          return <DogSummaryWrap onClick={() => setCurrent(key)} active={false} dog={dog} />;
+        })}
+      </DogWrapper>
+      <Button cursor={!(current === -1)} onClick={handleSubmit(current)}>
+        Submit
```

Comment:
```suggestion
        Confirm
```
Design says "Confirm"